### DISCUSSION TITLE
Watch for custom plugin configuration change

### DISCUSSION THREAD

**Discussion by aravindakumarbootlabs** [Kong/kong#9989] [Help]
I am writing a custom plugin, I want a callback to be called whenever there is change to the plugin configuration, how to do that?

**Comment by hanshuebner** [comment]
At this point, there is no separate callback for configuration changes.  You will have to manage change detection yourself by comparing the configuration provided to the `access` callback to a previous configuration that you have processed.  Let us know if you need further help with that.
